ydb/library/yql/udfs/common/knn/knn-enumerator.h

```diff
@@ -10,12 +10,11 @@ using namespace NYql::NUdf;
 
 template <typename TCallback>
 void EnumerateVector(const TUnboxedValuePod vector, TCallback&& callback) {
-    const auto elements = vector.GetElements();
+    const auto* elements = vector.GetElements();
     if (elements) {
-        const auto size = vector.GetListLength();
-        
-        for (ui32 i = 0; i < size; ++i) {
-            callback(elements[i].Get<float>());
+        const auto* end = elements + vector.GetListLength();
+        while (elements != end) {
+            callback(elements++->Get<float>());
```

Comment:
The difference is so vague.
Could you explain why you prefer `++` ?

Reply:
I think it's just my preference to iterator code, if needed I can revert this.

Reply:
I see. It's more C-like way.

To be more beautiful, let's use real C++ iterators inside range based for: `for (const auto& val : values)`.

Reply:
Ok, done